packages/dmn-js-drd/src/features/drill-down/DrillDown.js

```diff
@@ -87,6 +88,9 @@ export default class DrillDown {
     if (this._config.enabled !== false) {
       domClasses(html).add('interactive');
 
+      // Add tabindex attr. to allow keyboard tab focus on Drilldown button
+      domAttr(html, 'tabindex', '0');
```

Comment:
Adding the tabindex=0 attribute now allows the elements to be focused via tab key.

Reply:
This is not removed in latest commit, since the button element is being included in tab focus.